mobile/examples/phi-3/android/.gitignore

```
@@ -0,0 +1,11 @@
+*.iml
+.gradle
+/local.properties
+.idea
+.DS_Store
+/build
```

```suggestion
build
```

If we make this more generic we shouldn't need the .gitignore in the app directory